classes/item.rb

```diff
@@ -0,0 +1,21 @@
+
+class Item
+  attr_accessor :publish_date
+  attr_reader :genre, :label, :author, :archived, :source
+
+  def initialize(date)
+    @id = Random.rand(1..1000)
+    @publish_date = Date.strptime(date, '%Y-%m-%d')
```

Comment:
You should use the `require 'date'` syntax every time you implement the `date` method in your code.